tools/devctr/Dockerfile.aarch64

```diff
@@ -70,6 +70,7 @@ RUN apt-get update \
         libreadline-dev \
         libffi-dev \
         wget \
+        libbz2-dev \
```

Comment:
For what was this needed?

Reply:
It's needed for pandas to work on aarch64. It was already there in the x86_64 container but was missed in the aarch64 one.